docs/setup/discovery-server.md

```diff
@@ -0,0 +1,49 @@
+# Fast DDS Discovery Server
+
+This page gives instructions on how to use a Fast-DDS discovery server with the Create® 3 robot as a client.
```

Comment:
I just looked it up and apparently it's spelled "Fast DDS" and not "Fast-DDS." I have definitely made this mistake, before.